rules/windows/process_creation/win_susp_adfind.yml

```diff
@@ -19,7 +21,12 @@ logsource:
     service: process_creation
 detection:
     selection:
-        ProcessCommandline|contains: 'objectcategory'
+        ProcessCommandline|contains: 
+            - 'objectcategory'
+            - 'trustdmp'
+            - 'dcmodes'
+            - 'dclist'
+            - 'computers_pwdnotreqd'
         Image: 
             - '*\adfind.exe'
```

Comment:
Also please change this part of the code. Correct code:
`Image|endswith: '\adfind.exe'`
This will allow more detections systems to use this rule.